src/main/java/rails/game/specific/_18EU/StartCompany_18EU.java

```diff
@@ -180,7 +189,8 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
                 availableHomeStations.add(hex.getRelatedStop(stationId));
             }
         }
-        // selectedHomeStation is delayed
```

Comment:
did you check in a real game that this doesnt break the mechanismn ?

Reply:
What do you mean in a real game? I've been playing with this change for several days now-- since this only affects deserialization I would think that the loading and auto-loading I've been doing to play would validate?